projects/unix/Makefile

```diff
@@ -178,6 +171,19 @@ MKDIR ?= mkdir -p
 COMPILE.c = $(Q_CC)$(CC) $(CFLAGS) $(CPPFLAGS) $(TARGET_ARCH) -c
 LINK.o = $(Q_LD)$(CC) $(CFLAGS) $(LDFLAGS) $(TARGET_ARCH)
 
+ifeq ($(PIE), 1)
+  CFLAGS += -fPIE
+  LDFLAGS += -pie
+else
+  CFLAGS += -fno-PIE
+  ifeq ($(CC),$(CROSS_COMPILE)gcc)
+    PIE_DEFAULT = $(shell gcc -v 2>&1 | grep enable-default-pie)
```

Comment:
Nice but I would add a tiny comment: "check if PIE is the default for the compiler"

Reply:
done:)